### ISSUE TITLE
iOS Error Dialog - General handling question

### ISSUE THREAD

**Issue by SuperdanTX** [NativeScript/NativeScript#8214]
**Is your feature request related to a problem? Please describe.**
Let's say you have just boarded a plane, and switch to Airplane Mode. You head back to your awesome app written in Nativescript, and are greeted with an iOS generated message that is in a single white dialog that says "There is no Internet Connection". And there is no, "OK" or any other way to clear it. Your only remedy is to close the app and reopen it. The same thing would happen if an api call took long over cellular, "The request has timed out". Again, nice OS generated dialog that has locked the app.

**Describe the solution you'd like**
I'm not suggesting there is a fix, but wondering how others are handling this kind of behavior from the OS? Is there something we have completely missed to mitigate this sort of thing?

**Describe alternatives you've considered**
Hitting phone with hammer, joining a monastery, becoming an accountant.

**Additional context**
You guys are all awesome, and I appreciate your thoughts.
![IMG_0546](https://user-images.githubusercontent.com/40035221/71374766-888a5f00-2581-11ea-9cd3-892f6671ec2b.PNG)

**Comment by NickIliev** [comment]
@SuperdanTX you can use the native API to recreate an AlertView with no inte3raction options (as shown [here](https://stackoverflow.com/a/11261601/4936697)).

for example (TypeScript)
```TypeScript

let message = new UIAlertView();
message.title = "Hello World!";
message.message = "This is sample message.";
message.delegate = null;
message.cancelButtonTitle = null;
message.otherButtonTitles = null;

message.show();
```

Here is [a Playground demo](https://play.nativescript.org/?template=play-tsc&id=cyRx8X&v=3) that uses the NativeScript data conversion to im[lement the above (see the logic in the `onButtonTap` in the **_main-view-model.ts_**) - of course, you would need to provide your own logic for releasing the alert view (and/or other options)

**Comment by SuperdanTX** [comment]
Thank you Nick!

**Comment by NickIliev** [comment]
@SuperdanTX oh you mean that you are receiving an alert from the OS... I don't think you can handle messages that are coming from the system but it is also strange that you would receive a non-interactive dialog from the iOS- are you sure that those alerts are not shown by a third-party library/plugin?

**Comment by SuperdanTX** [comment]
Yep, it seems to be coming from the OS, particularly on iOS. We are poking around trying to find where it is coming from. Good idea to check third party plugins. I'll post back here once we spot it so any others that run into this may benefit.

**Comment by SuperdanTX** [comment]
Still testing for this. If I place the phone in Airplane mode, on our Nativescript app we get this:
![1](https://user-images.githubusercontent.com/40035221/72461216-21cd1f80-3794-11ea-895c-e658137666f3.png)

On another app, we get this, with an OK button.
![3](https://user-images.githubusercontent.com/40035221/72461261-390c0d00-3794-11ea-8fc1-17b4b9761466.png)

In our case, the only remedy is to close the app and reopen it.
We are at a loss as to why we don't get the OK button.
Nativescript Issue 4408 indicated this could be Confirms not having an OK button, while Dialogs always give an OK. Don't know if this is the issue or not, but we do use both in our app.

We will do some more testing and update.
